.github/workflows/docs.yml

```diff
@@ -0,0 +1,44 @@
+name: Docs
+on: [pull_request, push]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        pip install -e .[all]
```

Comment:
Hey, could you briefly explain how this works without installing SWIG? I'm actually wondering the same thing for Auto-sklearn right now...

Reply:
Swig is there, sorry for the confusion. It is needed when you run, and I have moved this line lower to clarify on this.